.travis.yml

```diff
@@ -1,7 +1,7 @@
 ---
 language: java
 jdk:
-  - oraclejdk8
+  - openjdk11
```

Comment:
This seems like it should be its own pull request.  It isn't related to the Hamming exercise, other than to allow Travis to pass, but this is more general than only Hamming.

Reply:
Ok, cool. Created a new PR, https://github.com/exercism/kotlin/pull/323 for changing the jdk only.